apps/example_apps/numbered_input_menu_test/main.py

```diff
@@ -0,0 +1,46 @@
+# coding=utf-8
+
+from apps.zero_app import ZeroApp
+from ui.numbered_menu import NumberedMenu
```

Comment:
You should add NumberedMenu import to `ui/__init__.py`, so that one can simply do `from ui import NumberedMenu`, like with other UI elements =)